modules/persistent-storage-csi-snapshots-create.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-snapshots.adoc
+
+[id="persistent-storage-csi-snapshots-create_{context}"]
+= Creating a volume snapshot
+
+Creation of a volume snapshot is triggered by the creation of a VolumeSnapshot API object.
+
+.Prerequisites
+* Logged in to a running {product-title} cluster.
+* An app; for example, `my-csi-app`.
+* A PVC pointing to the app; for example, `myclaim`. This is the PVC you will be making a snapshot of.
+* A storage class that is provisioning the storage backend.
+
+.Procedure
+
+To dynamically create a volume snapshot:
+
+. Save your volume snapshot class definition to a file.
+
++
+.volumesnapshotclass.yaml
+[source,yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass <1>
+metadata:
+  name: csi-hostpath-snap
+driver: hostpath.csi.k8s.io
+deletionPolicy: Delete
+----
+<1> Specifies the attributes that point to a CSI driver, such as HostPath.
++
+. Create the object definition file you saved in the previous step.
++
+----
+$ oc create -f volumesnapshotclass.yaml
+----
+
+. Create a VolumeSnapshot object:
+
++
+.volumesnapshot-dynamic.yaml
+[source,yaml]
+----
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshot <1>
+metadata:
+  name: mysnap
+spec:
+  volumeSnapshotClassName: csi-hostpath-snap <2>
+  source:
+    persistentVolumeClaimName: myclaim <3>
+----
+<1> A VolumeSnapshot object is automatically created based on the VolumeSnapshot.
```

Comment:
I'd remove this line. `VolumeSnapshot` is the API type of the object. This object (the VolumeSnapshot that was created by the user) will be updated once the `VolumeSnapshotContent` is created.